scripts/benchmark/go.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+commits=(main next)
+
+args=()
+
+for commit in "${commits[@]}"; do
+  rm -rf $commit
+  git -C ../.. archive $commit --prefix $commit/ | tar -x
+  (cd $commit; yarn; yarn build)
+  args+=("node ./bench.mjs $commit serial")
+  args+=("node ./bench.mjs $commit parallel")
+done
+
+hyperfine "${args[@]}"
```

Comment:
Maybe we can auto install hyperfine here https://github.com/prettier/prettier/blob/765cae25522f582b71a79e7fb10417eb4612abe6/.gitpod.yml#L4

So we can easily use gitpod to benchmark.

Reply:
I'm not sure that running benchmarks in the cloud in the cloud is a good idea, but we can give it a try. I never used gitpod. Feel free to add commits to this PR.